wgsl/index.bs

```diff
@@ -10418,11 +10777,11 @@ end of the loop body (CF_after_if).
 
 This example is modification of the [[#uniformity-example1|first example]], but
 uses a user-defined function call.
-The analysis tags both parameters of `scale` as
-[=ParameterRequiredToBeUniformForReturnValue=].
+The analysis tags the [=parameter return tag=] both parameters of `scale` as
```

Comment:
Nit: I think there's a missing "of" or something here.
"tags the parameter return tag [of?] both parameters "